py-polars/polars/testing/asserts/frame.py

```diff
@@ -48,6 +48,7 @@ def assert_frame_equal(
     check_exact
         Require data values to match exactly. If set to `False`, values are considered
         equal when within tolerance of each other (see `rtol` and `atol`).
+        Only takes effect for numeric, non-integer data types.
```

Comment:
Doesn't affect Decimals.

```suggestion
        Only affects columns with a Float data type.
```